### ISSUE TITLE
No style in the css for class="Grid-cell u-1of3"

### ISSUE THREAD

**Issue by getsetbro** [philipwalton/solved-by-flexbox#92]
Thanks for your work on this!

The demo page shows you can set classes on a Grid-cell like: `class="Grid-cell u-1of2"`
But the css linked on the page has no classes for that: https://github.com/philipwalton/solved-by-flexbox/blob/master/assets/css/components/grid.css
There only a class for a .Grid-cell in a container of "Grid--1of3":
```
.Grid--1of3 > .Grid-cell {
  flex: 0 0 33.3333%;
}
```

**Comment by philipwalton** [comment]
The other CSS you're referencing can be found in the [media.css](https://github.com/philipwalton/solved-by-flexbox/blob/867e92cfb8772db810df8f41bd785e7a9a235f27/assets/css/utils/media.css) ultility file. But I can see how this could be confusing. I'll look into adding extra links to these other files where it makes sense.

**Comment by getsetbro** [comment]
Would those files be combined if I cloned and built the repo?

**Comment by philipwalton** [comment]
Building the repo produces the files that are on https://philipwalton.github.io/solved-by-flexbox/, so yes.

**Comment by dafriend** [comment]
In the interests of BEM methodology I suggest refactoring u-* to Util-* or Utility-*

**Comment by philipwalton** [comment]
Technically the methodology used on this site is [SUIT CSS](https://suitcss.github.io/), which makes use of utility classes.

**Comment by rajasimon** [comment]
@philipwalton Also why `<div class="Grid-cell Grid--1of3">...</div>` is not working ?

**Comment by sanbor** [comment]
@rajasimon Probably because `Grid--1of3` is a modifier of `Grid`, not a modifier of `Grid-cell`.

**Comment by pReya** [comment]
I agree with @getsetbro. This confused me, too. I'd appreciate if you could include the full CSS in the examples on the page.

**Comment by gumatias** [comment]
Confusing indeed 😞
